# This example demonstrates the use of infinite retries for running
# the container to completion. It uses the `random-fail` container.
# For more details, see
# https://github.com/shrinandj/random-fail

apiVersion: argoproj.io/v1alpha1
kind: Workflow
metadata:
  generateName: container-retries-
spec:
  entrypoint: container-retries
  templates:
  - name: container-retries
    retryStrategy: {}
    container:
      image: shrinand/random-fail
      command: ["python"]
      args: ["/run.py", "40"]

